### PR TITLE
Add confirmation alert banner on successful save in EvaluationEditor and EvaluationRoundEditor (SWC-5384)

### PR DESCRIPTION
**Pull request by zimingd**

### CODE REVIEW

**Comment by zimingd** [comment]
Evaluation Editor:
image

Evaluation Rounds editor:
image

Evaluation Rounds editor with advanced limits expanded:
image

File: src/lib/containers/evaluation_queues/EvaluationCard.tsx [hunk 1]
```diff
@@ -118,15 +118,13 @@ export const EvaluationCard: React.FunctionComponent<EvaluationCardProps> = ({
...
-                <div>
-                  <Button
-                    className="submit-button"
-                    type="primary"
-                    onClick={onSubmit}
-                  >
-                    Submit
-                  </Button>
-                </div>
+                <Button
```
Comment (zimingd) [inline src/lib/containers/evaluation_queues/EvaluationCard.tsx:121]: was wrapping around `<div>` to get `display:block` behavior, now the button itself has `display:block`

File: src/__tests__/lib/containers/evaluation_queues/EvaluationEditor.test.tsx [hunk 2]
```diff
@@ -167,6 +169,27 @@ describe('test EvaluationEditor', () => {
     expect(mockUpdateEvaluation).toBeCalledWith(evaluation, sessionToken)
     expect(mockCreateEvaluation).not.toBeCalled()
     expect(mockOnSaveSuccess).toBeCalledWith(evaluationId)
+    expect(wrapper.find(ErrorBanner).exists()).toBe(false)
+    expect(wrapper.find('Alert.save-success-alert').exists()).toBe(true)
+  })
+
+  test('save button clicked - save failure', () => {
```
Comment (zimingd) [inline src/__tests__/lib/containers/evaluation_queues/EvaluationEditor.test.tsx:176]: added missing test when save fails

File: src/lib/containers/evaluation_queues/EvaluationEditor.tsx [hunk 3]
```diff
@@ -51,6 +51,13 @@ export const EvaluationEditor: React.FunctionComponent<EvaluationEditorProps> =
   }
 
   const [error, setError] = useState<SynapseClientError>()
+  const [showSaveSuccess, setShowSaveSuccess] = useState<boolean>(false)
+
+  useEffect(() => {
```
Comment (zimingd) [inline src/lib/containers/evaluation_queues/EvaluationEditor.tsx:56]: useful for when other REST calls, such as DELETE fail

File: src/lib/style/components/evaluation_queue/_created-on.scss [hunk 4]
```diff
@@ -1,6 +1,6 @@
 .created-on {
   // verticlally center the user profile pic to be in line with the other text
-  display: inline-flex;
+  display: flex;
```
Comment (zimingd) [inline src/lib/style/components/evaluation_queue/_created-on.scss:3]: previously misunderstood `flex` vs `inline-flex`. I actually wanted block behavior

**Review by jay-hodgson — APPROVED** [review]

**Pull request merged**